[2024-04-02T21:59:14.708+0800] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: product_etl_dag.extract_RawEmployee_load.gcp_load_employee scheduled__2024-04-01T09:00:00+00:00 [queued]>
[2024-04-02T21:59:14.717+0800] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: product_etl_dag.extract_RawEmployee_load.gcp_load_employee scheduled__2024-04-01T09:00:00+00:00 [queued]>
[2024-04-02T21:59:14.717+0800] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-04-02T21:59:14.727+0800] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): extract_RawEmployee_load.gcp_load_employee> on 2024-04-01 09:00:00+00:00
[2024-04-02T21:59:14.732+0800] {standard_task_runner.py:60} INFO - Started process 24190 to run task
[2024-04-02T21:59:14.735+0800] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'product_etl_dag', 'extract_RawEmployee_load.gcp_load_employee', 'scheduled__2024-04-01T09:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/var/folders/9z/pmblcfnj66jbv4s_wjl_bmx80000gn/T/tmprm3xglgl']
[2024-04-02T21:59:14.739+0800] {standard_task_runner.py:88} INFO - Job 17: Subtask extract_RawEmployee_load.gcp_load_employee
[2024-04-02T21:59:14.800+0800] {task_command.py:423} INFO - Running <TaskInstance: product_etl_dag.extract_RawEmployee_load.gcp_load_employee scheduled__2024-04-01T09:00:00+00:00 [running]> on host kalers-mbp
[2024-04-02T21:59:17.061+0800] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='product_etl_dag' AIRFLOW_CTX_TASK_ID='extract_RawEmployee_load.gcp_load_employee' AIRFLOW_CTX_EXECUTION_DATE='2024-04-01T09:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-01T09:00:00+00:00'
[2024-04-02T21:59:17.062+0800] {logging_mixin.py:188} INFO - Data load error: [Errno 2] No such file or directory: '/Users/kalerbramastha/Library/CloudStorage/OneDrive-Personal/career_assessment/akasia_de/Scripts/ETL, Data Warehouse and Analytics Task/SA.json'
[2024-04-02T21:59:17.063+0800] {python.py:202} INFO - Done. Returned value was: None
[2024-04-02T21:59:17.065+0800] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=product_etl_dag, task_id=extract_RawEmployee_load.gcp_load_employee, execution_date=20240401T090000, start_date=20240402T135914, end_date=20240402T135917
[2024-04-02T21:59:17.115+0800] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-02T21:59:17.121+0800] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
